reference/api/multi_search.mdx

```diff
@@ -7,6 +7,10 @@ description: The /multi-search route allows you to perform multiple search queri
 
 The `/multi-search` route allows you to perform multiple search queries on one or more indexes by bundling them into a single HTTP request. Multi-search is also known as federated search.
 
+## Example application
+
+Meilisearch CRM SaaS demo uses multi-search to allow searching through contacts, deals, and companies via a unified search bar. Check out the [SaaS demo](https://saas.meilisearch.com/?utm_campaign=oss&utm_source=docs&utm_medium=multi-search-api) and the [Github repository](https://github.com/meilisearch/saas-demo/).
+
```

Comment:
Could we skip this one? I really think it's a bit out of place to have it in a reference page, where users come to get hard data on how to use the API.

```suggestion
```

Reply:
Do you have another page to recommend to include this link?

It's the only one I found that focuses on multi-search